src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,149 @@
+package wolox.training.models;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    private String genre;
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private String pages;
+
+    @Id
```

Comment:
Change the @id annotation of the isbn attribute. Put in this attribute a @Column(nullable = false, unique = true) annotation.